test/ui-automation/test/helpers/wallet.js

```diff
@@ -18,8 +18,12 @@ let signedUpUserEmail;
 exports.signUp = async ({ email }) => {
   // login and consent
   await _getSignUp(email);
-  // register chapi
-  await allow();
+
+  if (browser.config.walletName === "wallet.trustbloc.local:8091") {
```

Comment:
Rather than this, can we pass a bool to this function to skip this part ? This way we can keep helpers cleaner. The bool param can from calling function. Also, lets not hardcode the value in the code.